ydb/docs/en/core/yql/reference/syntax/show_create.md

```diff
@@ -0,0 +1,126 @@
+# SHOW CREATE
+
+`SHOW CREATE` returns a query (possibly consisting of several DDL statements) required to recreate the structure of the specified object: a [table](../../../concepts/datamodel/table.md) or [view](../../../concepts/datamodel/view.md).
+
+## Syntax
+
+```yql
+SHOW CREATE [TABLE|VIEW] <name>;
+```
+
+## Parameters
+
+* `TABLE|VIEW` — The object type. Valid values are `TABLE` or `VIEW`.
+* `<name>` — The object name. An absolute path may also be specified.
+
+## Result
+
+The command always returns **exactly one row** with three columns:
+
+| Path            | PathType   | CreateQuery                      |
+|-----------------|------------|----------------------------------|
+| Path            | Table/View | DDL statements for creation      |
+
+- **Path** — The path to the object (e.g., `MyTable` or `MyView`).
+- **PathType** — The type of object: `Table` or `View`.
+- **CreateQuery** — The complete set of DDL statements needed to recreate the object:
```

Comment:
As far as I understand, the "statements needed to recreate" claim doesn't seem to be fulfilled because the returned queries lack a `DROP TABLE ... IF EXISTS` (and similar) statements, and thus they would fail if someone tried to execute them.